pennychat/serializers.py

```diff
@@ -1,15 +1,63 @@
 from rest_framework import serializers
-
-from .models import PennyChat, FollowUp
+from rest_framework.fields import Field
+from pennychat.models import (
+    PennyChat,
+    FollowUp,
+    Participant,
+)
 from users.serializers import UserProfileSerializer
 
 
+class ChoiceField(Field):
```

Comment:
Annoying that the choice field didn't work, but I'm glad you were able to create a custom one. I wonder if this belongs in a different place though since we might reuse it across apps. Not super important right now and we can always come back to it later.

Reply:
I'm going to keep it here until we find we need it later. My hope is that we actually figure out what the problem was and just get back to using the real `ChoiceField`

Reply:
That reminds me that I wanted to open an issue about that on the DRF repo.